common/tasks/interleaved_weighted_round_robin.go

```diff
@@ -399,7 +419,7 @@ LoopDispatch:
 				numThrottled++
```

Comment:
If all channels are throttled, wouldn't this be a busy loop?

Reply:
If all channels are throttled, the function will return on L413.